docs/src/local_development.md

```diff
@@ -0,0 +1,46 @@
+# Local Development Instructions
+
+To install the `ethstaker-deposit-cli`, follow these steps:
+
+## Prerequisites
+
+Ensure you have the following software installed on your system:
+
+- **Git**: Version control system to clone the repository. [Download Git](https://git-scm.com/downloads)
+- **Python 3.12+**: The programming language required to run the tool. [Download Python](https://www.python.org/downloads/)
```

Comment:
I wonder if we should relax this in the documentation and everywhere in the repository. We do support Python 3.9+ through the CI and many OSes or distro already come with a 3.9 >= Python version < 3.12 installed. What do you think?